src/coreclr/debug/ee/functioninfo.cpp

```diff
@@ -1576,15 +1574,11 @@ DebuggerJitInfo *DebuggerMethodInfo::FindOrCreateInitAndAddJitInfo(MethodDesc* f
 
     if (startAddr == NULL)
     {
-        // This will grab the start address for the current code version.
         startAddr = g_pEEInterface->GetFunctionAddress(fd);
```

Comment:
This will only end up calling `MethodDesc::GetNativeCode` and not `MethodDesc::GetNativeCodeAnyVersion` which is essentially what this path was doing. Is there any reason to change this?

Reply:
When you say "Is there any reason to change this?", is 'this' referring to lines 1583-1587 below? Or are you expecting that line 1580 should have changed and you are questioning why it didn't?

Assuming you are asking why 1583-1587 changed it is because we eliminated all code paths that would have reached 1583 except the case where the method is not yet jitted. This part of the change is cleanup to demonstrate the call to GetNativeCodeReJITAware() is not relevant here any longer. I don't expect anything would fail if 1583-1587 was left untouched and the definition of GetNativeCodeReJITAware() was preserved.